samples/openthread/cli/README.rst

```diff
@@ -339,6 +339,113 @@ To test the Thread 1.2 features, complete the following steps:
    .. note::
         The DUA registration with the Backbone Router is not yet supported.
 
+#. Link Metrics
+
+   * Perform a Link Metrics query (Single Probe).
+
+      ``ot linkmetrics query \<ipaddr\> single [pqmr]``
```

Comment:
I would avoid including OT CLI help here. We already have a link to OpenThread CLI Reference in this page.